src/filters/log.rs

```diff
@@ -156,11 +156,8 @@ impl<'a> Info<'a> {
     }
 
     /// View the host of the request
-    pub fn host(&self) -> Option<&str> {
-        self.route
-            .headers()
-            .get(header::HOST)
-            .and_then(|v| v.to_str().ok())
+    pub fn host(&self) -> Option<Authority> {
```

Comment:
So as to not be a breaking change, can this keep returning `&str`?

Reply:
I think it can. However then:

  - It's inconsistent with `warp::host()`
  - As a user, it's nice that you're made aware that it can also contain port (high potential of biting you)
  - As a user, it's nice that you don't have to parse into `Authority` yourself (esp. since it already was `Authority` internally)

I would just bite the bullet if we can and go for the cleaner API. Otherwise I can change it, yes.